examples/sklearn-logreg-mnist/utils.py

```diff
@@ -0,0 +1,74 @@
+from typing import Tuple, Union, List
+import numpy as np
+from sklearn.linear_model import LogisticRegression
+import openml
+
+XY = Tuple[np.ndarray, np.ndarray]
+Dataset = Tuple[XY, XY]
+LogRegParams = Union[XY, Tuple[np.ndarray]]
+XYList = List[XY]
+
+
+def get_model_parameters(model: LogisticRegression) -> LogRegParams:
+    """Returns the paramters of a sklearn LogisticRegression model"""
+    if model.fit_intercept:
+        params = (model.coef_, model.intercept_)
+    else:
+        params = (model.coef_,)
+    return params
+
+
+def set_model_params(
+    model: LogisticRegression, params: LogRegParams
+) -> LogisticRegression:
+    """Sets the parameters of a sklean LogisticRegression model"""
+    model.coef_ = params[0]
+    if model.fit_intercept:
+        model.intercept_ = params[1]
+    return model
+
+
+def set_initial_params(model: LogisticRegression):
+    """
+    # Sets initial parameters as zeros
+    # Required for since model params are uninitialized until model.fit is called
+    # But server asks for inital parameters from clients at launch
+    # Refer to sklearn.linear_model.LogisticRegression documentation
+    # for more information.
```

Comment:
```suggestion
    Sets initial parameters as zeros
    Required for since model params are uninitialized until model.fit is called
    But server asks for initial parameters from clients at launch
    Refer to sklearn.linear_model.LogisticRegression documentation
    for more information.
```